### DISCUSSION TITLE
Shouldn't it say "applied knowledge" instead of "practical steps"?

### DISCUSSION THREAD

**Discussion by bittner** [evildmp/diataxis-documentation-framework#45] [Ideas]
The [Diátaxis framework slide](https://diataxis.fr/_images/diataxis.png) categorizes Tutorials and How-To Guides as "practical steps". That seems like a non-academic label for what – in terms of study subjects or similar – would be called "applied", I believe.

### Examples

- Theoretical Computer Science ([example](https://www.sciencedirect.com/journal/theoretical-computer-science), [Wikipedia](https://en.wikipedia.org/wiki/Theoretical_computer_science))
- Theoretical Mathematics ([example](https://math.asu.edu/research/theoretical-mathematics), [example](https://admission.wsu.edu/academics/fos/Public/field.castle?id=1680))
- Applied Mathematics ([example](https://www.epfl.ch/education/master/programs/applied-mathematics/), [Wikipedia](https://en.wikipedia.org/wiki/Applied_mathematics))
- Applied Sciences ([example](https://www.scimagojr.com/journalsearch.php?q=21100829268&tip=sid), [Wikipedia](https://en.wikipedia.org/wiki/Applied_science))

Would it make sense to use the term "Applied knowledge" (or something similar) in the slide instead?

**Comment by evildmp** [comment]
It's a good question. In fact it's not really a distinction between the theoretical and the applied, or even between theory and practice. It's a distinction between different things the documentation contains, i.e. what it has to offer the user.

How-to guides and tutorials both contain steps for the user to follow. In the case of tutorials, it's to develop practical skill and theoretical knowledge and understanding *by following the practical steps it describes". In a tutorial, the user may have barely any knowledge of the subject to apply. 

A how-to guide also describes practical steps, though the purpose is to accomplish a work task, not to educate the user.

As far as the documentation is concerned, the two key distinctions are that the things on the "theoretical side" can be contained inside the mind, while the ones on the other side cannot because they belong to the hands; and that the ones on the first side can be expressed outside time, while the ones on the other express sequences in time.

The problematic term in the distinction for me is actually the word "knowledge". I don't like the idea that documentation "contains knowledge". Knowledge is something that minds contain. (This is especially so in the case of understanding.) However so far I haven't come up with a compact and intuitive form of words in English. 

For a mind, theoretical (roughly, "knowing that") and practical (roughly, "knowing how") knowledge are bound up with each other, and I don't believe that we can place them into neat boxes. All of skill-related documentation applies to them both. This is a central tenet of this framework, which holds that we only learn by doing. 

Consider terms like episteme, gnosis, phronesis, praxis, etc - they are all related to the categories here, but they don't fit them in any neat ways. That's because the framework isn't really based on different conceptions of knowledge, but on what the user is doing and needs at different times.
